src/entities/createPool/types.ts

```diff
@@ -180,4 +181,17 @@ export type CreatePoolLiquidityBootstrappingInput = Omit<
 > & {
     lbpParams: LBPParams;
     poolType: PoolType.LiquidityBootstrapping;
+    poolCreator?: Address;
 };
+
+export type LBPMigrationParams = {
+    bptLockDuration: bigint;
```

Comment:
Would be useful to change to something that describes time unit? e.g. `bptLockDurationMinutes`

Reply:
I have changed the lock duration to now be `bptLockDurationinSeconds`. About start time and endtime I could see it be changed to `startTimestamp` and `endTimestamp`